File: c++/1.1 Herencia Simple y Modificadores de acceso protegidos/1.1.1 Herencia/NominaV4/NominaTest.cpp
#include <iostream>
#include "Persona.h"
#include "Gerente.h"
#include "Vendedor.h"
#include "VendedorEventual.h"

using namespace std;

int main(int argc, char const *argv[])
{
  Vendedor vendedor = Vendedor("Julio Mancera Castro", 32, 1.73f, "Electrónica");
  cout << vendedor.toString() << endl;

  Gerente gerente = Gerente("Maribel Estrada Contreras", 34, 1.70f, "Chalco");       
  cout << gerente.toString() << endl;

  VendedorEventual vendedorE = VendedorEventual("Rosa Castro Luján", 25, 1.75f, "Ropa", 200.0f);
	cout << vendedorE.toString() << endl;
  
  return 0;
}

File: python/1.1 Herencia Simple y Modificadores de acceso protegidos/1.1.1 Herencia/NominaV4/NominaTest.py
# -*- coding: utf-8 -*-
from VendedorEventual import VendedorEventual
from Vendedor import Vendedor
from Gerente import Gerente

# Uso de la clase Vendedor
if __name__ == "__main__":
    vendedor = Vendedor("Julio Mancera Castro", 32, 1.73, "Electrónica")
    print(vendedor)
    gerente = Gerente("Maribel Estrada Contreras", 34, 1.70, "Chalco")
    print(gerente)
    vendedorE = VendedorEventual("Rosa Castro Luján", 25, 1.75, "Ropa", 200.0)
    print(vendedorE)
